src/plugins/data/public/search/expressions/es_raw_response.test.ts

```diff
@@ -0,0 +1,827 @@
+                    tax_amount: 0,
+                    product_id: 18495,
+                    category: "Women's Shoes",
+                    sku: 'ZO0376403764',
+                    taxless_price: 49.99,
+                    unit_discount_amount: 0,
+                    min_price: 25,
+                    _id: 'sold_product_562351_18495',
+                    discount_amount: 0,
+                    created_on: '2016-12-09T19:55:12+00:00',
+                    product_name: 'Ankle boots - cognac',
+                    price: 49.99,
+                    taxful_price: 49.99,
+                    base_unit_price: 49.99,
+                  },
+                  {
+                    base_price: 28.99,
+                    discount_percentage: 0,
+                    quantity: 1,
+                    manufacturer: 'Tigress Enterprises',
+                    tax_amount: 0,
+                    product_id: 22598,
+                    category: "Women's Clothing",
+                    sku: 'ZO0050800508',
+                    taxless_price: 28.99,
+                    unit_discount_amount: 0,
+                    min_price: 14.78,
+                    _id: 'sold_product_562351_22598',
+                    discount_amount: 0,
+                    created_on: '2016-12-09T19:55:12+00:00',
+                    product_name: 'Shift dress - black',
+                    price: 28.99,
+                    taxful_price: 28.99,
+                    base_unit_price: 28.99,
+                  },
+                ],
+                sku: ['ZO0376403764', 'ZO0050800508'],
+                taxful_total_price: 78.98,
+                taxless_total_price: 78.98,
+                total_quantity: 2,
+                total_unique_products: 2,
+                type: 'order',
+                user: 'wilhemina',
+                geoip: {
+                  country_iso_code: 'MC',
+                  location: {
+                    lon: 7.4,
+                    lat: 43.7,
+                  },
+                  continent_name: 'Europe',
+                  city_name: 'Monte Carlo',
+                },
+                event: {
+                  dataset: 'sample_ecommerce',
+                },
+              },
+              fields: {
+                order_date: ['2020-07-10T19:55:12.000Z'],
+                'products.created_on': ['2016-12-09T19:55:12.000Z', '2016-12-09T19:55:12.000Z'],
+              },
+            },
+          ],
+        },
+        aggregations: {
+          '2': {
+            doc_count_error_upper_bound: 0,
+            sum_other_doc_count: 0,
+            buckets: [
+              {
+                key: 'FEMALE',
+                doc_count: 1033,
+              },
+              {
+                key: 'MALE',
+                doc_count: 944,
+              },
+            ],
+          },
+        },
+      },
+    };
+    const result = esRawResponse.to!.datatable(response, {});
+    expect(result).toMatchSnapshot();
```

Comment:
This test isn't testing what the title says, that docs are returned instead of aggs. Can you add some assertions?

Reply:
`'returns docs if both docs and aggs are present on response'` so yes, the docs were expected, however i am changing this as per your suggestion to return aggs if both are present